docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in [this issue](https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
 
-See Repository Examples:
+
+In the repo we also have two examples showing how Sagas can be tested:
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/counter/test/sagas.js
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/test/sagas.js
+
+
+
+***External Resources***
+
+Redux Saga Test Plan - https://github.com/jfairbank/redux-saga-test-plan
+
+From the README:
```

Comment:
It might not be the most ideal solution - but given that there's still an open discussion (in the issue I linked to) about the best approach to testing sagas, I think this is at least an improvement on the previous version of this page in the docs, which I found to be insulting and unhelpful.

I think if there was a agreement on the best way to test sagas, this page of the docs would describe that approach in enough detail for someone to implement tests for their sagas. Even if it described the approach used in the examples that would be better than what's currently there.

In the meantime this adds just a little bit of help to developers who are hoping to get some information on how to test their sagas.

As for the including a snippet of the README, again it's about being helpful to developers - if redux-saga-test had a more detailed README I would have used it. The README snippet gives the developer some minimal context before investigating either of the repos, again it's about having something nicer than just a page with a few links on.